src/python/bot/fuzzers/libfuzzer.py

```diff
@@ -334,19 +337,20 @@ class FuchsiaQemuLibFuzzerRunner(new_process.ProcessRunner, LibFuzzerCommon):
   def __init__(self, executable_path, default_args=None):
     fuchsia_pkey_path = environment.get_value('FUCHSIA_PKEY_PATH')
     fuchsia_portnum = environment.get_value('FUCHSIA_PORTNUM')
-    if not fuchsia_pkey_path or not fuchsia_portnum:
+    fuchsia_resources_dir = environment.get_value('FUCHSIA_RESOURCES_DIR')
+    if (not fuchsia_pkey_path or not fuchsia_portnum or
+        not fuchsia_resources_dir):
       raise fuchsia.errors.FuchsiaConfigError(
-          'FUCHSIA_PKEY_PATH and/or FUCHSIA_PORTNUM was not set')
-    # yapf: disable
-    self.ssh_args = [
-        '-vvv',
-        '-i', fuchsia_pkey_path,
-        '-o', 'StrictHostKeyChecking no',
-        '-o', 'UserKnownHostsFile=/dev/null',
-        '-p', str(fuchsia_portnum),
-        'localhost'
-    ]
-    # yapf: enable
+          ('FUCHSIA_PKEY_PATH, FUCHSIA_PORTNUM, or FUCHSIA_RESOURCES_DIR was '
+           'not set'))
+    self.host = Host.from_dir(
+        os.path.join(fuchsia_resources_dir, 'build', 'out', 'default'))
+    self.device = Device(self.host, 'localhost', fuchsia_portnum)
+    pkg, tgt = environment.get_value('FUZZ_TARGET').split('/')
```

Comment:
s/pkg/package/ and s/tgt/target/

Reply:
Also, what's the format of "FUZZ_TARGET" ? I'm assuming `package_name/binary_name` ?

@inferno-chromium Seems like another case where we need to support `/` in a fuzz target name. Seems like we need to normalize the binary name too (and not just project).

Reply:
Yep, that's the expected format of FUZZ_TARGET.

Completed s/pkg/package/ and s/tgt/target/.

(Leaving comment open--I don't have context on the question of "supporting slashes in target names" but I assume @inferno-chromium may need to peek :) )